test/BUILD.gn

```diff
@@ -949,9 +949,6 @@ if (!is_android) {
       "//brave/browser/tor",
       "//brave/browser/ui",
       "//brave/browser/ui/tabs",
-      "//brave/common:network_constants",
```

Comment:
The files in the component are simple enough to not split them into a several source_sets.
=> all targets merged into the one to simplify using the component.